test/OrchardCore.Tests/Localization/LocalizationManagerTests.cs

```diff
@@ -67,11 +72,49 @@ public void GetDictionarySelectsPluralRuleFromProviderWithHigherPriority()
                 It.IsAny<CultureDictionary>())
             );
 
-            var manager = new LocalizationManager(new[] { _pluralRuleProvider.Object, highPriorityRuleProvider.Object }, new[] { _translationProvider.Object }, _memoryCache);
+            var manager = new LocalizationManager([_pluralRuleProvider.Object, highPriorityRuleProvider.Object], [_translationProvider.Object], _memoryCache);
 
             var dictionary = manager.GetDictionary(CultureInfo.GetCultureInfo("cs"));
 
             Assert.Equal(dictionary.PluralRule, csPluralRuleOverride);
         }
+
+        [Theory]
+        [InlineData("en", "Hello en !")]
+        [InlineData("zh-CN", "你好！")]
```

Comment:
```suggestion
        [InlineData("zh-CN", "你好")]
```

Reply:
In Chinese, we use full-width symbols ∶)